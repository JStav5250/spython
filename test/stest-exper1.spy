int x = 3

if bool(x):
	type(x)
