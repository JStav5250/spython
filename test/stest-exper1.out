TypeError: cannot cast operand of typ 'int' to type 'bool'!!
